src/CMakeLists.txt

```diff
@@ -1,19 +1,41 @@
 
 cmake_minimum_required(VERSION 3.1)
 
-project(Iaito VERSION 1.0.0)
+project(iaito VERSION 1.0.0)
 
-set(CMAKE_CXX_STANDARD 11)
+set(QT_MIN_VERSION "5.6.0")
 
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+include(CheckCXXCompilerFlag)
+include(FeatureSummary)
+include(GenerateExportHeader)
+
+find_package(Qt5 ${QT_MIN_VERSION} NO_MODULE REQUIRED
+	WebEngine
+	WebEngineWidgets
+)
+
+find_package(Radare2 1.4.0 REQUIRED)
+include_directories(${RADARE2_INCLUDE_DIRS})
+
+feature_summary(WHAT ALL FATAL_ON_MISSING_REQUIRED_PACKAGES)
 
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
+set(CMAKE_INSTALL_PREFIX ${BUILD_DIR})
+set(BIN_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/bin)
```

Comment:
Yes this a dirty hack I had to set it because install targets wont work without it. Also I mentioned clearly in PR this in WIP